doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md

```diff
@@ -0,0 +1,89 @@
+# ADR-0001: Update the InVEST SDR LS Factor
+
+Author: James
+Science Lead: Rafa
+
+## Context
+
+Since we released the updated InVEST SDR model in InVEST 3.1.0, we have seen a
+common refrain of users and NatCap science staff noticing that the LS factor
+output of SDR did not produce realistic results and that the LS factor produced
+by SAGA was much more realistic.  We have over the years made a couple of notable
+changes to the model and to the LS factor that have altered the output including:
+
+1. The SDR model's underlying routing model was changed from d-infinity to MFD in 3.5.0
+2. The $x$ parameter was changed in InVEST 3.8.1 from the true on-pixel aspect
+   $|\sin \theta|+|\cos \theta|$ (described in Zevenbergen & Thorne 1987 and repeated
+   in Desmet & Govers 1996) to the weighted mean of proportional flow from the
+   current pixel to its neighbors.
+3. A typo in a constant value in the LS factor was corrected in InVEST 3.9.1
+4. An `l_max` parameter was exposed to the user in InVEST 3.9.1
+
+Despite these changes to the LS factor, we still received occasional reports
+describing unrealistic LS factor outputs from SDR and that SAGA's LS factor
+was much more realistic.
+
+After diving into the SAGA source code, it turns out that there are several
+important differences between the two despite both using Desmet & Govers (1996)
+for their LS factor equations:
+
+1. The contributing area $A_{i,j-in}$ is not strictly defined in Desmet &
+   Govers (1996), it is only referred to as "the contributing area at the inlet
+   of a grid cell with coordinates (i, j) (m^2)".
+   InVEST assumes that "contributing area" is $area_{pixel} \cdot n\_upstream\_pixels$.
```

Comment:
On inspecting the rich diff, it looks like there might be some syntax issues around subscripting. I noticed a few subscripts that didn't look as expected.